src/synthcity/metrics/eval.py

```diff
@@ -175,17 +185,32 @@ def evaluate(
                 continue
             if metric.name() not in metrics[metric.type()]:
                 continue
-            scores.queue(
-                metric(
-                    reduction=reduction,
-                    n_histogram_bins=n_histogram_bins,
-                    task_type=task_type,
-                    random_state=random_state,
-                    workspace=workspace,
-                ),
-                X_gt.sample(eval_cnt),
-                X_syn.sample(eval_cnt),
-            )
+            if X_augmented and "augmentation" in metric.name():
+                scores.queue(
+                    metric(
+                        reduction=reduction,
+                        n_histogram_bins=n_histogram_bins,
+                        task_type=task_type,
+                        random_state=random_state,
+                        workspace=workspace,
+                        use_cache=use_cache,
+                    ),
+                    X_gt.sample(eval_cnt),
+                    X_augmented.sample(eval_cnt),
```

Comment:
Is `sample(eval_cnt)` relevant for the augmented dataset? The augmented dataset will be larger than `X_gt` everytime, right?